README.md

```diff
@@ -132,3 +140,8 @@ If you need to make sure if your latest code works with sample project or your e
 ./gradlew clean uploadArchives
 mvn deploy -Dmaven.test.skip=true
 ```
+
+### install local
```

Comment:
I don't want to have two ways to build the core library. If we can install the lib locally via Gradle, that'll be nice.